### ISSUE TITLE
possible to morph images with preparation?

### ISSUE THREAD

**Issue by yairEO** [jembezmamy/morpher-js#14]
I have a website with thousands of photos (hosted on 3rd party service) and I want to morph between 2 random photos. is this possible?

**Comment by jembezmamy** [comment]
Well, you'll need to describe the geometry of these photos first. Take a look at the [GUI](http://jembezmamy.github.io/morpher-js/) – this should give you the general idea, how this geometry should be defined. You can define it programmatically as described in the last section on [demos page](http://jembezmamy.github.io/morpher-js/demos.html). MorpherJS won't guess this geometry for you. You have to define it or you can use some other library to generate it automatically (e.g. by face recognition).
If you have geometry data for each picture, then it should be easy to pick just two random and morph them.

**Comment by yairEO** [comment]
yeah well, since these images I want to morph are randomly chosen from the internet, I never know what they would be. it's on-the-fly. Do you really think there is a javascript program which could generate the geometry automatically using face recognition? Do you know of any?

Thank you so much for answering!
I will be using this for such an amazing project, people's jaws would drop.

**Comment by jembezmamy** [comment]
I don't know if there is any JS face recognition library, try searching the internet. 😄 Some people use MorpherJS with a server side face recognition, which is probably faster than JavaScript one.

**Comment by yairEO** [comment]
even if I searched the internet, I thought that those programs must output the same data format which your program takes as input, and I thought it would take forever to find such a program and it is worth saving time by asking you, since you obviously know about this subject more than me :)

Anyway, GOOD WORK! it's a real cool thing you've made.

**Comment by jembezmamy** [comment]
Oh, I'm almost sure you won't find a program that outputs data in the same format MorpherJS uses. I've just made this format up to suit my needs.
